source/administration/config-settings.rst

```diff
@@ -4788,6 +4798,19 @@ Data Prefetch
 | This feature's ``config.json`` setting is ``"ExperimentalDataPrefetch": true`` with options ``true`` and ``false``.                                                  |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable File Search
+^^^^^^^^^^^^^^^^^^
+
+This configuration setting enables users to search documents attached to messages by filename. To enable users to search documents by their content, you must also enable the ``ExtractContent`` configuration setting. See our `Enable Document Search by Content <https://docs.mattermost.com/administration/config-settings.html#enable-document-search-by-content>`__ documentation for details. Document content search is available in Mattermost Server from v5.35, with mobile support coming soon. 
+
+**True**: Supported document types are searchable by their filename. 
```

Comment:
The "supported" string is not needed here, actually, all files can be search by filename, the is not restriction in the format of the file itself.

Reply:
Thanks!